### ISSUE TITLE
Startup warnings

### ISSUE THREAD

**Issue by SilleBille** [SilleBille/pki#442]
This issue was migrated from [Pagure Issue #2986](https://pagure.io/dogtagpki/issue/2986).Originally filed by cheimes on 2018-04-11

 On Fedora 28, I'm seeing two warnings in PKI's journald log for FreeIPA's Dogtag instance. One is about a missing jar file. The second warning is about an unknown property.

```
Started PKI Tomcat Server pki-tomcat.
Java virtual machine used: /usr/lib/jvm/jre-1.8.0-openjdk/bin/java
classpath used: /usr/share/tomcat/bin/bootstrap.jar:/usr/share/tomcat/bin/tomcat-juli.jar:/usr/lib/java/commons-daemon.jar
main class used: org.apache.catalina.startup.Bootstrap
flags used: -DRESTEASY_LIB=/usr/share/java/resteasy -Djava.library.path=/usr/lib64/nuxwdog-jni
options used: -Dcatalina.base=/var/lib/pki/pki-tomcat -Dcatalina.home=/usr/share/tomcat -Djava.endorsed.dirs= -Djava.io.tmpdir=/var/lib/pki/pki-tomcat/temp -Djava.util.logging.config.file=/var/lib/pki/pki-tomcat/conf/logging.properties -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager     -Djava.security.manager     -Djava.security.policy==/var/lib/pki/pki-tomcat/conf/catalina.policy
arguments used: start
WARNING: Problem with JAR file [/usr/share/pki/server/common/lib/symkey.jar], exists: [false], canRead: [false]
WARNING: [SetPropertiesRule]{Server/Service/Engine/Host/Valve} Setting property 'resolveHosts' to 'false' did not find a matching property.
```

Installed versions:
```
# rpm -qa | grep ^pki-
pki-kra-10.6.0-0.3.fc28.noarch
pki-tools-10.6.0-0.3.fc28.x86_64
pki-server-10.6.0-0.3.fc28.noarch
pki-base-10.6.0-0.3.fc28.noarch
pki-ca-10.6.0-0.3.fc28.noarch
pki-base-java-10.6.0-0.3.fc28.noarch
```

**Comment by SilleBille** [comment]
Posted by edewata on 2018-04-12: 

https://tomcat.apache.org/tomcat-8.5-doc/config/valve.html

> resolveHosts: This attribute is no longer supported. Use the connector attribute enableLookups instead.

https://tomcat.apache.org/tomcat-8.5-doc/config/http.html

> enableLookups: Set to true if you want calls to request.getRemoteHost() to perform DNS lookups in order to return the actual host name of the remote client. Set to false to skip the DNS lookup and return the IP address in String form instead (thereby improving performance). By default, DNS lookups are disabled.

**Comment by SilleBille** [comment]
Posted by edewata on 2018-04-25: 

The first warning will be fixed in ticket #2972.

**Comment by SilleBille** [comment]
Posted by edewata on 2018-04-25: 

The second warning is fixed in master branch:

* https://github.com/dogtagpki/pki/commit/f0d608338d6cdee67405c92d721c2797ac4d5ca2

The remaining task is to remove resolveHosts from existing instances with an upgrade script.
